packages/auth/amplify_auth_cognito_dart/lib/src/sdk/src/cognito_identity/cognito_identity_client.dart

```diff
@@ -1,18 +1,4 @@
-// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Do we want/need a license header?

Reply:
I think we decided for generated code to not have it since it would just be overwritten and complicated the generation process.

This was just a remnant from before that decision I think.